decidim-proposals/spec/events/decidim/proposals/proposal_endorsed_event_spec.rb

```diff
@@ -9,7 +9,7 @@
   let(:resource) { proposal }
   let(:author) { create :user, organization: proposal.organization }
 
-  let(:extra) { { endorser: author } }
+  let(:extra) { { endorser_id: author.id } }
```

Comment:
Maybe the endorser shouldn't be the author of the proposal? (I know this was already like this but it's better to fix it)

Reply:
It is not, it's the author of the endorsement (I got confused too)

Reply:
Oh, I guess it's because the event is an `Authorable`